### ISSUE TITLE
Webform duplicate submission error messages

### ISSUE THREAD

**Issue by bewoldt** [uiowa/uiowa#5129]
# Problem
When submitting a form without filling out required fields, a panel shows up at the top of the page telling you how many errors there are on the form and links you to the required fields. When adding a form to a block using layout builder, this duplicates the error messages.

Example form in a block. Submitting will create duplicate error messages.
https://studenthealth.uiowa.edu/info/above-beyond-nomination-form

Example form - direct link to form. No duplicates when submitting.
https://studenthealth.uiowa.edu/form/above-beyond-nomination-form

<img width="872" alt="Screen Shot 2022-04-12 at 11 44 43 AM" src="https://user-images.githubusercontent.com/731952/163013318-55e230c6-f616-44e1-b52d-73e324dd1320.png">

**Comment by joewhitsitt** [comment]
I wonder if this is related to https://github.com/uiowa/uiowa/issues/2579.

**Comment by bspeare** [comment]
@joewhitsitt adjusting validation options on `/admin/structure/webform/config` seems to fix this issue.  Can you remind me why we aren't using the default tooltip error messages?

<img width="759" alt="Screenshot 2023-03-21 at 2 14 03 PM" src="https://user-images.githubusercontent.com/1036433/226717126-54781536-d3a6-45ec-8631-60e1d7927e89.png">

<img width="837" alt="Screenshot 2023-03-21 at 2 13 14 PM" src="https://user-images.githubusercontent.com/1036433/226717172-5ca68b32-eca3-45ce-89e3-e8a78c7a1039.png">

**Comment by joewhitsitt** [comment]
>Can you remind me why we aren't using the default tooltip error messages?

For webforms? I am not sure. Early D8 we had this drupal developer named @briand44  ;) who did the original evaluation of webform and proposed most of the defaults.

The IFE module is turned on for the profile and I do recall it seemed to work in some spots and not in others in terms of accessibility (announcing or with the right amount of context), but I don't think it has been revisited since.

**Comment by briand44** [comment]
Yeah, I could be totally off on this but I thought way back in the day there were accessibility issues with the inline validation stuff. That is the only reason I can think off on why we would have intentionally went a different direction here.

**Comment by joewhitsitt** [comment]
Though IFE may solve the visual issue here, I think https://github.com/uiowa/uiowa/issues/2579 is very much the underlying issue that should be solved.

**Comment by bspeare** [comment]
Thanks for chiming in on this.  I'd love to revisit the IFE since that looks like a great user experience compared to what we currently have implemented.
